example/spec/parallel/ingest/ingestFromPdrSpec.js

```diff
@@ -81,51 +82,58 @@ describe('Ingesting from PDR', () => {
   let testDataFolder;
   let testSuffix;
   let workflowExecution;
+  let addedCollection;
 
   beforeAll(async () => {
-    config = await loadConfig();
-
-    process.env.ExecutionsTable = `${config.stackName}-ExecutionsTable`;
-    process.env.PdrsTable = `${config.stackName}-PdrsTable`;
-
-    executionModel = new Execution();
-
-    const testId = createTimestampedTestId(config.stackName, 'IngestFromPdr');
-    testSuffix = createTestSuffix(testId);
-    testDataFolder = createTestDataPath(testId);
-
-    pdrFilename = `${testSuffix.slice(1)}_${origPdrFilename}`;
-
-    provider = { id: `s3_provider${testSuffix}` };
-
-    // populate collections, providers and test data
-    await Promise.all([
-      updateAndUploadTestDataToBucket(
-        config.bucket,
-        s3data,
-        testDataFolder,
-        [
-          { old: 'cumulus-test-data/pdrs', new: testDataFolder },
-          { old: 'DATA_TYPE = MOD09GQ;', new: `DATA_TYPE = MOD09GQ${testSuffix};` }
-        ]
-      ),
-      uploadTestDataToBucket(
-        config.bucket,
-        unmodifiedS3Data,
-        testDataFolder
-      ),
-      addCollections(config.stackName, config.bucket, collectionsDir, testSuffix, testId),
-      addProviders(config.stackName, config.bucket, providersDir, config.bucket, testSuffix)
-    ]);
-
-    // Rename the PDR to avoid race conditions
-    await s3().copyObject({
-      Bucket: config.bucket,
-      CopySource: `${config.bucket}/${testDataFolder}/${origPdrFilename}`,
-      Key: `${testDataFolder}/${pdrFilename}`
-    }).promise();
-
-    await deleteS3Object(config.bucket, `${testDataFolder}/${origPdrFilename}`);
+    try {
```

Comment:
Most of the changes here are adding beforeAllFailed to various setup steps, excepting the collection addition on L163